cartoframes/data/services/bq_datasets.py

```diff
@@ -183,6 +183,10 @@ def status(self):
 
             body = response.json()
 
+            if body['status'] == 'failure':
+                msg = u'Client Error: %s' % (body['errors'][0])
```

Comment:
We usually use `format`:
msg = 'Client Error: {}'.format(body['errors'][0])

Reply:
we are using it in that way across the whole module. Don't you mind open an issue separately to achieve that?